### PR TITLE
Feat/#19 format support

### PR DESCRIPTION
**Pull request by benjamin-allion**

### CODE REVIEW

File: lib/core/formats.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const { logger } = require('../../config/logger');
+
+const FormatTypes = {
+  UPPERCASE_FORMAT: 'uppercase',
```
Comment (RayedB) [inline lib/core/formats.js]:
```suggestion
  UPPERCASE: 'uppercase',
```

File: lib/core/formats.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  LOWERCASE_FORMAT: 'lowercase',
```
Comment (RayedB) [inline lib/core/formats.js]:
```suggestion
  LOWERCASE: 'lowercase',
```

We are already in a FormatType Object, I don't believe we should add "Format" at the end of every key.
As in JavaScript it is toLowerCase I think "LOWERCASE" or "UPPERCASE" is enough

**Review by RayedB — APPROVED** [review]

File: lib/core/formats.js (hunk 2 above)
Comment (benjamin-allion) [reply lib/core/formats.js]: 👍

File: lib/core/formats.js (hunk 1 above)
Comment (benjamin-allion) [reply lib/core/formats.js]: 👍

**Pull request merged**